src/syscheckd/registry/registry.c

```diff
@@ -185,7 +185,36 @@ int fim_registry_validate_path(const char *entry_path, const registry *configura
  * @return A fim_registry_key object holding the information from the queried key, NULL on error.
  */
 fim_registry_key *fim_registry_get_key_data(HKEY key_handle, const char *path, const registry *configuration) {
-    return NULL;
+    fim_registry_key *key;
```

Comment:
Set path and arch values for the registry key inside this function.

Reply:
Done 7968a28